servers/rendering/renderer_viewport.cpp

```diff
@@ -1138,6 +1138,7 @@ void RendererViewport::viewport_set_screen_space_aa(RID p_viewport, RS::Viewport
 void RendererViewport::viewport_set_use_taa(RID p_viewport, bool p_use_taa) {
 	Viewport *viewport = viewport_owner.get_or_null(p_viewport);
 	ERR_FAIL_COND(!viewport);
+	ERR_FAIL_COND_EDMSG(OS::get_singleton()->get_current_rendering_method() != "forward_plus", "TAA is only available when using the Forward+ renderer");
```

Comment:
Tiny stylistic change:

```suggestion
	ERR_FAIL_COND_EDMSG(OS::get_singleton()->get_current_rendering_method() != "forward_plus", "TAA is only available when using the Forward+ renderer.");
```

Reply:
Will add the period. 

We don't want to just warn, TAA is not supported in either of the backends, but this setting changes control flow in RendererSceneCull, so we need to tell the user that its not supported with a toast, and exit the function.